LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2008, Aldo Cortesi. All rights reserved.
+Copyright (c) 2025, Aldo Cortesi. All rights reserved.
```

Comment:
I think this should really read 2008-2025; but while you're here, can you change change the copyright holder too?

```
Copyright (c) 2008-2025, Qtile Contributors. All rights reserved.
```